web-platform/website/src/components/pages/login/EmailSignupView.tsx

```diff
@@ -98,6 +102,10 @@ export function EmailSignupView() {
 
     try {
       await signupWithEmail(emailRef.current, passwordRef.current);
+      if (installedStudyIds.length < 1 && isV2Enabled) {
+        setLoginState(LoginState.GetExtension);
+        localStorage.setItem("isExtensionView", "isExtensionView");
```

Comment:
We don't need local storage. If we want to see that extensions are installed we can query installStudyIds like we are doing here and in new login page. That way we don't depend on local storage.